tools/UpdatePaths/Scripts/27268_snake_case_remap.txt

```diff
@@ -25,7 +25,7 @@
 /obj/item/gun/projectile/automatic/pistol/APS/@SUBTYPES : /obj/item/gun/projectile/automatic/pistol/aps/@SUBTYPES{@OLD}
 /datum/mapGeneratorModule/border/syndieWalls/@SUBTYPES : /datum/map_generator_module/border/syndie_walls/@SUBTYPES{@OLD}
 /mob/living/simple_animal/pet/dog/corgi/Lisa/@SUBTYPES : /mob/living/simple_animal/pet/dog/corgi/lisa/@SUBTYPES{@OLD}
-/mob/living/simple_animal/mouse/white/Brain/@SUBTYPES : /mob/living/simple_animal/mouse/white/brain/@SUBTYPES{@OLD}
+/mob/living/simple_animal/mouse/white/Brain/@SUBTYPES : /mob/living/basic/simple_animal/white/brain/@SUBTYPES{@OLD}
```

Comment:
Don't change old update paths. Create a new update paths please.
```suggestion
/mob/living/simple_animal/mouse/white/Brain/@SUBTYPES : /mob/living/simple_animal/mouse/white/brain/@SUBTYPES{@OLD}
```
and in a new file
```suggestion
/mob/living/simple_animal/mouse/@SUBTYPES : /mob/living/basic/mouse/@SUBTYPES{@OLD}
```